client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -150,6 +150,22 @@ export class PluginMeta extends Component {
 		);
 	};
 
+	renderSupportedFlag() {
+		const supportedAuthors = [ 'Automattic', 'WooCommerce' ];
+		if (
+			this.props.isJetpackSite ||
+			supportedAuthors.indexOf( this.props.plugin.author_name ) < 0
```

Comment:
Absolutely not a blocker, but I tend to like the more recent array prototype methods for things like this just because they read a little more "narratively":

```
if (
			this.props.isJetpackSite ||
			!supportedAuthors.find(a => a === this.props.plugin.author_name)
)
```